samples/ingress/database-api/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
```

Comment:
💡 This is interesting, there was a `dotnet6` upgrade PR in last release, so is this something missed or `3.0` was in user for some special case. cc: @hsubramanianaks

Reply:
this is in samples/todo-app which we use extensively for testing. Actual docker images for devhost, restoration and routing manager were updated to dotnet 6. :)